### PR TITLE
Email course alerts

### PR DESCRIPTION
**Pull request by ryandv**
Woo! #124 is finally done. Unfortunately our version of font-awesome is too old to have the [filled-in bell icon](http://fortawesome.github.io/Font-Awesome/3.2.1/icon/bell-alt/) so I've just used an [X](http://fortawesome.github.io/Font-Awesome/3.2.1/icon/remove/) icon. Upgrading to FontAwesome 3.2.1 started screwing around with a bunch of other icons on the page (shortlist buttons), so for now I've decided to leave things as is and just get the core functionality shipped.

There are some transient issues (`Object [object Object] has no method 'on'`) with the profile page related to jQuery and Bootstrap but I am having trouble consistently reproducing them - if they surface on anyone else's machine, please let me know.

### CODE REVIEW

File: models/course_alert.py [hunk 1]
```diff
@@ -170,3 +172,50 @@ def send_alert(self, sections):
...
+
+    def send_alert(self, sections):
+
+        _conn = boto.connect_ses(
+            aws_access_key_id=s.AWS_KEY_ID,
+            aws_secret_access_key=s.AWS_SECRET_KEY)
+
+        current_user = user.User.objects.get(id=self.user_id)
+
+        email_body = \
```
Comment (NorthStarPolaris) [inline models/course_alert.py]: Does it make sense to hard-code the email body? Why not read it from a static file and fill in the parameters? That way, we could use a templating engine to fill in the email body.

File: models/user.py [hunk 2]
```diff
@@ -304,6 +310,9 @@ def last_schedule_paste(self):
     def get_user_courses(self):
```
Comment (NorthStarPolaris) [inline models/user.py]: Wouldn't it make rename this method to ```user_courses``` and put the method under an ```@property``` decorator? The method doesn't take in any parameter except ```self```.

**Comment by JGulbronson** [comment]
@ryandv still working on this?

**Comment by ryandv** [comment]
No. Merge conflict broke my implementation.

**Pull request closed without merging**